src/platform/STM32/timer_stm32h7xx.c

```diff
@@ -148,15 +148,16 @@ const timerHardware_t fullTimerHardware[FULL_TIMER_CHANNEL_COUNT] = {
     DEF_TIM(TIM14, CH1N, PF9, 0, 0, 0),
 
 // Port H
-// Port H is not available for LPQFP-100 or 144 and TFBGA-100 package
-//    DEF_TIM(TIM12, CH1, PH6, 0, 0, 0),
-//    DEF_TIM(TIM12, CH2, PH9, 0, 0, 0),
-//    DEF_TIM(TIM5, CH1, PH10, 0, 0, 0),
-//    DEF_TIM(TIM5, CH2, PH11, 0, 0, 0),
-//    DEF_TIM(TIM5, CH3, PH12, 0, 0, 0),
-//    DEF_TIM(TIM8, CH1N, PH13, 0, 0, 0),
-//    DEF_TIM(TIM8, CH2N, PH14, 0, 0, 0),
-//    DEF_TIM(TIM8, CH3N, PH15, 0, 0, 0),
```

Comment:
please retain comments if live timers will not be included

Reply:
Hmm I want to just update this from the generator to include all of the options, however the TIMER_PIN_MAP index relies on the order of these definitions. This means we would need to ensure the order doesn't change when autogenerating (which of course it does).

Reply:
okay, thank you. i am not privy to the technicals.

Reply:
Instead of leaving them commented out I just added those options to the list